movement/movement.h

```diff
@@ -295,4 +295,6 @@ void movement_play_alarm(void);
 
 uint8_t movement_claim_backup_register(void);
 
+#define MOVEMENT_SETTINGS_FILE "settings.bin"
```

Comment:
Don't know if there's a better place for this or a better way than a define, but a `const char*` wasn't working well